wrapper/fmriprep_docker.py

```diff
@@ -245,11 +246,21 @@ def main():
     g_dev.add_argument('-p', '--patch-nipype', metavar='PATH',
                        type=os.path.abspath,
                        help='working nipype repository')
+    g_dev.add_argument('--fs-license', metavar='PATH', type=os.path.abspath,
+                       default=os.getenv('FS_LICENSE', None),
+                       help='folder containing FreeSurfer\'s license.txt file')
```

Comment:
While it's a little bit neither here nor there, I think this is more of a "wrapper" option than a "dev" option.